diag_manager/diag_manager.F90

```diff
@@ -1675,7 +1675,9 @@ LOGICAL FUNCTION diag_send_data(diag_field_id, field, time, is_in, js_in, ks_in,
     CHARACTER(len=128) :: error_string, error_string1
 
     REAL, ALLOCATABLE, DIMENSION(:,:,:) :: field_out !< Local copy of field
-    class(*), pointer, dimension(:,:,:,:) :: field_modern !< i8 4d remapped pointer
+    class(*), pointer, dimension(:,:,:,:) :: field_modern !< 4d remapped pointer
+    logical,  pointer, dimension(:,:,:,:) :: mask_modern  !< 4d remapped pointer
+    class(*), pointer, dimension(:,:,:,:) :: rmask_modern !< 4d remapped pointer
```

Comment:
I don't like using time-relative names such as "modern" for a variable that is permanent.  `field_remap` or `field_pointer`, or `field_p` would be better

Reply:
Updated to `*_remap`